build/docker/edgecore.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM kubeedge/build-tools:latest as builder
+ARG ARCH=amd64
+WORKDIR /work
+ADD . .
+RUN mkdir -p bin &&\
+    if [ "${ARCH}" = "amd64" ]; then make WHAT=edgecore && cp _output/local/bin/edgecore bin/edgecore ; \
+    elif [ "${ARCH}" = "arm64" ]; then make crossbuild ARM_VERSION=GOARM8 WHAT=edgecore && cp _output/local/bin/edgecore bin/edgecore ; \
+    elif [ "${ARCH}" = "arm" ]; then make crossbuild ARM_VERSION=GOARM7 WHAT=edgecore && cp _output/local/bin/edgecore bin/edgecore;  \
```

Comment:
Now the build-tools base has supported multi-CPU architecture, so no need to do the crossbuild here :)

Reply:
Any comments about this?